.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[alias]
+backend="build --no-default-features"
```

Comment:
Inconsistency. It is better to make all these short commands either for build or for run, but not mixed. It is up to you to decide what is better here

Reply:
Or if you need to mix them, then give the commands clearer names. For example, `backend_build`, `frontend_run`, ...

Reply:
IMHO, the ideal way is to have the following commands:
1. `cargo build_backend` - builds backend only
2. `cargo build_frontend` - builds frontend only
3. `cargo build` - builds both backend and frontend and copy frontend output to backend directory with static files
4. `cargo run_backend` - runs backend only (only as api without frontend static files)
5. `cargo run` - builds and runs both backend and frontend

In all this cases swagger should be enabled by default